src/main/java/com/dreammaster/scripts/ScriptAdvancedSolarPanel.java

```diff
@@ -31,65 +31,91 @@ public void loadRecipes() {
 
         addShapedRecipe(
                 getModItem(AdvancedSolarPanel.ID, "advanced_solar_helmet", 1, 0, missing),
-                new Object[] { "craftingToolWrench",
-                        getModItem(AdvancedSolarPanel.ID, "BlockAdvSolarPanel", 1, 0, missing),
-                        "craftingToolScrewdriver", "circuitAdvanced",
-                        getModItem(IndustrialCraft2.ID, "itemArmorNanoHelmet", 1, 32767, missing), "circuitAdvanced",
-                        "cableGt04Aluminium", getModItem(GregTech.ID, "gt.blockmachines", 1, 22, missing),
-                        "cableGt04Aluminium" });
+                "craftingToolWrench",
+                getModItem(AdvancedSolarPanel.ID, "BlockAdvSolarPanel", 1, 0, missing),
+                "craftingToolScrewdriver",
+                "circuitAdvanced",
+                getModItem(IndustrialCraft2.ID, "itemArmorNanoHelmet", 1, 32767, missing),
```

Comment:
This one and two more in this file.

Reply:
oh ye i just replaced these in RecipeRemovals file